PythonSetupForIFTPipeline.jl/README.md

```diff
@@ -0,0 +1,28 @@
+# Python Setup for IFTPipeline.jl
+
+This project is designed to initialize the Python Environment for IFTPipeline.jl.
+It should be used once before IFTPipeline.jl is instantiated.
+It will:
+
+- Create a new `base` Conda environment which PyCall will use,
+- Install dependencies to that environment,
+- Rebuild PyCall using the updated environment.
+
+Since the `base` environment is shared between all PyCall instances by default, it only needs to be run if the Conda environment is:
+- missing – as in a new installation, or 
+- broken – new incompatible dependencies have been added.
+
+The goal is to remove the need for this subpackage at some point in future. 
+
+## User Guide
+
+To use, run:
+```bash
+julia --project="PythonSetupForIFTPipeline.jl" PythonSetupForIFTPipeline.jl/setup.jl
+```
+
+> If this fails, try removing the existing Conda installation for Julia:
+> ```bash
+> rm -r ~/.julia/conda/
+> ```
+> ... and then retry.
```

Comment:
I think `make` can make these commands more abstract/simpler.

Reply:
I'm not sure how `make` would make this simpler. Can you help me understand?

It would certainly hide the single line of code you'd need to copy, but it might encourage us to make `make` the standard for setting up the whole environment. I don't think that will help anyone – overall it'll add another level of abstraction which isn't necessary. `makefiles` are notorious for being difficult to understand. 

Ideally, we wouldn't need this crutch at all, and we'll just fix the python dependencies at some point.